tools/acorn-optimizer.mjs

```diff
@@ -978,67 +979,84 @@ function emitDCEGraph(ast) {
 // way (and we leave further DCE on the JS and wasm sides to their respective
 // optimizers, closure compiler and binaryen).
 function applyDCEGraphRemovals(ast) {
-  const unused = new Set(extraInfo.unused);
+  const unused_exports = new Set(extraInfo.unused_exports);
```

Comment:
```suggestion
  const unusedExports = new Set(extraInfo.unusedExports);
```
I think this would be more consistent, though I don't feel strongly. I can see an argument for the JSON style being different than JS.